app/helpers/orders_helper.rb

```diff
@@ -45,6 +45,6 @@ def contact_info(contact_address)
   end
 
   def contact_name(contact)
-    [contact['FirstName'], contact['LastName']].reject(&:empty?).join(' ')
+    [contact['FirstName'], contact['LastName']].reject(&:empty?).join(' ') unless contact.nil?
```

Comment:
Within `app/views/orders/show.html.erb` the method `contact_info` passes a local variable to the 'contact_information' partial. There are a number of calls in there that use nested hash calls, so it seems like those would throw an error. Do we have a test that tests for this? I presume our tests have a contact person so we'll not see this in our tests, we should add one.